content/pages/pricing.html

```diff
@@ -2,389 +2,263 @@
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">1 day support response time</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">6 concurrent builds</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Search term analytics</div>
-                    <div class="description">90 days</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Pageview analytics</div>
-                    <div class="description">90 days</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Auth auditing</div>
-                    <div class="description">90 days</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Single sign-on (SSO)</div>
-                    <div class="description">with Google</div>
-                  </div>
-                </div>
-              </div>
-
-            </div>
-          </div>
         </div>
-        <!-- end card -->
-
       </div>
+    </div>
+  {% endblock pricing_community_intro %}
+
+  {% block pricing_business_header %}
+    <div class="ui padded container">
+      <h1 class="ui medium header">
+        Plan pricing
+        <span class="sub header">
+          Read the Docs plan features and pricing
+        </span>
+      </h1>
+    </div>
+  {% endblock pricing_business_header %}
 
-
-      <div class="ui segment basic vertical padded center aligned">
-        <p>
-          <a class="ui button orange tiny"  data-analytics="commercial-signup" href="https://readthedocs.com/accounts/signup">Sign up</a>
-          for a business account.
-        </p>
+  <div class="ui padded container">
+    <div class="ui huge secondary pointing stackable menu" data-module="tabmenu" data-module-history="true">
+      <div class="center menu">
+        <a class="violet active item" data-tab="business">Read the Docs for Business</a>
+        <a class="teal item" data-tab="community">Read the Docs Community</a>
       </div>
+    </div>
+    <div class="ui horizontally fitted basic active tab segment" data-tab="business">
+      <div class="ui centered three column tablet stackable grid">
+        <div class="stretched row">
+
+          {% call pricing.plan(
+            name="Basic plan",
+            price="$50",
+            color="violet",
+          ) %}
+            <div class="ui list relaxed divided">
```

Comment:
We lost the little copy that we have about who the plan is good for (eg `For public-facing docs on your domain`). The feature listing mostly covers this, but I do think these short snippets are useful to add back to contextualize the plans.

Reply:
Yeah, I tried to fit these in but they made these lists more cluttered. I did like the addition in theory though, but perhaps I'll put testing these out again back on my later list at #167